### DISCUSSION TITLE
Customize pages in the admin panel

### DISCUSSION THREAD

**Discussion by tamer-yigit** [OrchardCMS/OrchardCore#16772] [Q&A]
Hello,

I want to make appearance changes on the pages (content) that I added to the admin panel.

These changes, for example, on the screen where the pages I added are listed;
- Creating a new column between the title and the buttons
- Adding various icons or buttons in this column and redirecting them to certain actions when clicked.

In order to make such a development, do I need to add an Admin Theme to the Project in Visual Studio and override it with shapes from there?
Or is there a way to do these via the Orchard admin panel without adding any themes?
I have watched many videos on this topic and I am confused.
Can you help? Thanks.

**Comment by hishamco** [comment]
- if you are on the code-base you can change whatever you want
- if you are using OC NuGet packages you could create a theme or from within the module you need to override the views or create shapes to override what you want

Have a look at [Orchard Core Contrib](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules) I create a theme to change the login page and some of the look and feel. Also you can check the existing module to see some shapes that injected into the OC UI

**Comment by tamer-yigit** [comment]
I have attached a screenshot of what I want to do. I have added a content called Person. And I want to make the screen where they are listed like the attached image. Is there a way to do this via Admin Panel? If I have to do it via Visual Studio, what will happen to the other screens that I do not want to change the appearance of? All their appearances such as HTML, CSS are lost. In this case, should I prepare a separate shape for each of them?

![image](https://github.com/user-attachments/assets/6b1535e7-9aea-4757-a516-930b307e997a)

**Comment by ns8482e** [comment]
Each row is rendered by SummaryAdmin shape, you can override summary admin shape for your content type eg Content-Page.SummaryAdmin.cshtml
